tests/nnfw_api/src/one_op_tests/Conv2D.test.cc

```diff
@@ -276,3 +276,29 @@ TEST_F(GenModelTest, neg_OneOp_Conv2D_I8_NonZero_ZeroPoints)
 
   SUCCEED();
 }
+
+TEST_F(GenModelTest, neg_OneOp_Conv2D_I8_Hybrid_PerLayer)
+{
+  CircleGen cgen;
+  std::vector<int8_t> weight_data{1, 2, 3, 4, 5, 6, 7, 8, 9};
+  uint32_t weight_buf = cgen.addBuffer(weight_data);
+  std::vector<float> bias_data{0, 2, 4};
+  uint32_t bias_buf = cgen.addBuffer(bias_data);
+  int in = cgen.addTensor({{1, 1, 1, 3}, circle::TensorType::TensorType_FLOAT32});
+  // Hybrid does not support per-layer.
```

Comment:
What is `per-layer`?

Reply:
I realized that it's per-tensor.
(Optional) It would be better to change it to per-tensor.

Reply:
I used per-layer because `circle-quantizer` uses the term in source and command line argument. I am okay for both term. But I will change as you suggested.

Reply:
It's just optional. The term is already in use, so I think it's better to consider changing the term layer.